pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -410,6 +410,7 @@ func (k *Kubernetes) ConfigVolumes(name string, service kobject.ServiceConfig) (
 			if err != nil {
 				return nil, nil, nil, errors.Wrap(err, "k.CreatePVC failed")
 			}
+			log.Printf("volume %q of size \"100 Mi\" created, PersistentVolume should be created before in order to PersistentVolumeClaim start working, unless it will be in pending state \n If Cluster has dynamic Provisioning and Storage Classes, So you don't have to do anything", volumeName)
```

Comment:
This should be `INFO` and this output is *wayyyyy* too long. 

We already output when a PersistentVolumeClaim is created. This should be a small INFO.